test/Interop/Cxx/class/inheritance/Inputs/functions.h

```diff
@@ -7,32 +7,48 @@ struct NonTrivial {
   NonTrivial() {}
   ~NonTrivial() {}
 
-  inline const char *inNonTrivial() const { return "NonTrivial::inNonTrivial"; }
-  inline const char *inNonTrivialWithArgs(int a, int b) const {
+  inline const char *inNonTrivial() const
+      __attribute__((swift_attr("import_unsafe_api"))) {
```

Comment:
Review note: I was originally update all these tests to be safe, but I decided that should happen in a follow up PR. These are testing very specific things, so we should upgrade them slowly/incrementally to ensure we don't regress.